x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/X-ConstantVector.java.st

```diff
@@ -31,7 +31,11 @@ $endif$
 
     Constant$Type$Vector($type$ value, int positionCount, BlockFactory blockFactory) {
         super(positionCount, blockFactory);
+$if(BytesRef)$
+        this.value = BytesRef.deepCopyOf(value);
```

Comment:
I wonder if we're better off copying on the caller. I imagine there are lots of folks who already deep copy.

Reply:
Updated